.github/ci.sh

```diff
@@ -163,6 +163,13 @@ zip_dist() {
   name="${name:-"cryptol-$VERSION-$RUNNER_OS-x86_64"}"
   cp -r dist "$name"
   tar -cvzf "$name".tar.gz "$name"
+  sname="${name}-with-solvers"
```

Comment:
For the sake of separating concerns, I think it would be better to split `zip_dist` into two functions, one for creating the vanilla bindist and another for creating the `with-solvers` bindist. This has the benefit of making it easier to create and test each bindist locally without having to create the other, and it more closely mirrors the structure of `ci.yml`, which signs and uploads each bindist separately.

Reply:
Yeah, that's a good thought. I'll change it.